SPECS/binutils/binutils.spec

```diff
@@ -124,19 +101,29 @@ make %{?_smp_mflags} check
 %{_includedir}/dis-asm.h
 %{_includedir}/bfd_stdint.h
 %{_includedir}/diagnostics.h
+%{_includedir}/ctf-api.h
+%{_includedir}/ctf.h
 %{_libdir}/libbfd.a
 %{_libdir}/libopcodes.a
 %{_libdir}/libbfd.so
 %{_libdir}/libopcodes.so
+%{_libdir}/bfd-plugins/libdep.so
+%{_libdir}/libctf-nobfd.a
+%{_libdir}/libctf-nobfd.so*
+%{_libdir}/libctf.a
+%{_libdir}/libctf.so*
```

Comment:
Small request: could you have separate entries for [libname].so and [libname].so.[major_version_numer]*? This will help to keep track of any version updates modifying library versions, so we know when to update dependent packages.

Reply:
Sure, updated